### ISSUE TITLE
Julia meetup CPH (denmark) oktober 12

### ISSUE THREAD

**Issue by mkborregaard** [JuliaCPH/Meetups.jl#1]
Moved from my unrelated repo:

I (@1oly @andreasnoack @cth @dhoegh @emilbayes @eugene @eveydee @kkholst @martinandersen @mkborregaard @pkofod @robertdj @simulkade @stemann @ulfworsoe) er alle danskere / folk der bor i DK der har github contributions til pakker i Julia's metadata.

Vi prøver at starte et julia-meetup i Danmark. Der er et møde næste torsdag i København. Der er et event at tilmelde sig her: https://www.meetup.com/JuliaCPH/events/243862923/
Og i øvrigt en #danmark kanal på julia's slack :-)

Håber at se mange af jer!

**Comment by dhoegh** [comment]
Fedt initiativ, jeg bor i Aalborg så jeg kommer ikke til at deltage grundet lang transporttid.

**Comment by pkofod** [comment]
> Fedt initiativ, jeg bor i Aalborg så jeg kommer ikke til at deltage grundet lang transporttid.

Det er fair nok. Skriv endelig hvis du er i København og vil snakke Julia! Og husk at finde os på slack https://slackinvite.malmaud.com/ hvis du er til den slags!

**Comment by robertdj** [comment]
Det lyder cool med Julia-meetup i DK! Men jeg bor også i Aalborg, så det bliver ikke denne gang...
Jeg kendte ikke den slack-kanal, men har sendt en forespørgsel.

**Comment by pkofod** [comment]
> Det lyder cool med Julia-meetup i DK! Men jeg bor også i Aalborg, så det bliver ikke denne gang...
> Jeg kendte ikke den slack-kanal, men har sendt en forespørgsel.

Nej, slacken er lidt svær at finde hvis man ikke så beskeden på discourse, men jeg tror der arbejdes på at gøre den mere synlig.

**Comment by andreasnoack** [comment]
Den er på https://julialang.org/community/ nu, men det er stadig ikke det
mest synlige sted

**Comment by emilbayes** [comment]
Godt initiativ! Meetup'et var mega hygge og Plots var et spændende emne. Diskussionen efter var også rigtig god, så jeg vil helt sikkert se frem til næste meetup 👍

**Comment by pkofod** [comment]
> Godt initiativ! Meetup'et var mega hygge og Plots var et spændende emne. Diskussionen efter var også rigtig god, så jeg vil helt sikkert se frem til næste meetup :+1:

Godt at høre! Enig I at det var hyggeligt, og hvis visualisering(ssoftware) ikke er noget der kan diskuteres, så ved jeg ikke hvad kan! Fedt at få forskellige indspark.

**Comment by pkofod** [comment]
Hvis nogle er interesserede så er der et nyt meetup 20. november, se #2 .

Husk at følge enten meetup.com gruppen, JuliaCPH twitter profilen https://twitter.com/juliacph, eller "Watch" denne gruppe.
